src/main/java/appeng/client/gui/implementations/GuiCraftConfirm.java

```diff
@@ -210,6 +210,8 @@ private boolean isSimulation() {
         return ((ContainerCraftConfirm) this.inventorySlots).isSimulation();
     }
 
+    final ReadableNumberConverter converter = ReadableNumberConverter.INSTANCE;
```

Comment:
This seems to be redundant if you use java.text.NumberFormat directly

Reply:
I would concur but for some reason removing it and using NumberFormat directly caused the game to crash. It was written like this before.

Reply:
Really? What is in the crash log?